cli/pcluster/cw-dashboard-substack.cfn.yaml

```diff
@@ -0,0 +1,253 @@
+{%- set coord = {'x': 0, 'y': 0} %}
+{%- macro update_position(dx, dy) %}
+  {%- if coord.update({'x': (coord.x+dx)}) %} {% endif %}
+  {%- if coord.x > 18 %}
+    {%- if coord.update({'x': 0, 'y': coord.y+dy}) %} {% endif %}
+  {%- endif %}
+{%- endmacro %}
+{%- macro update_pos_after_section(dy) %}
+  {%- if coord.update({'x': 0, 'y': coord.y+dy}) %} {% endif %}
+{%- endmacro %}
+Parameters:
+  # Cluster parameters
+  PclusterStackName:
+    Description: Name of the cluster to which this dashboard belongs
+    Type: String
+  # EC2 parameters
+  MasterInstanceID:
+    Description: ID of the Master instance
+    Type: AWS::EC2::Instance::Id
+  # EBS parameters
+  EBSVolumesIDs:
+    Description: IDs of the EBS volumes used
+    Type: CommaDelimitedList
+  # RAID parameters
+  RAIDVolumesIDs:
+    Description: Volume IDs of the resulted RAID EBS volumes
+    Type: CommaDelimitedList
+  # EFS parameters
+  EFSFileSystemID:
+    Description: ID of the EFS volume used
+    Type: String
+  # FSx parameters
+  FSXFileSystemId:
+    Description: ID of the FSx volume used
+    Type: String
+Conditions: {}
+Resources:
+  MasterCloudWatchDashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+      DashboardName: !Join
+        - '-'
+        - - !Ref 'PclusterStackName'
+          - 'Master'
+      DashboardBody: !Join
+        - ''
+        - - '{"widgets":['
+          - !Join # TODO: delete this join (since the one before is the same) if we do not change it back
+            - '' # Do not use comma join: Would make a problem when using If statements as Join(',', ['a','','b'])='a,,b'
+            - # Head Node Instance Metrics
+              - '{"type":"text","x":{{ coord.x }},"y":{{ coord.y }},"width":24,"height":1,"properties":{"markdown":"\n#
+                      Head Node Instance Metrics\n"}}'
+              {%- set ec2_metrics = [["CPUUtilization"],["NetworkPacketsIn", "NetworkPacketsOut"], ["NetworkIn", "NetworkOut"],
+                                     ["StatusCheckFailed", "StatusCheckFailed_Instance", "StatusCheckFailed_System"],
+                                     ["DiskReadBytes", "DiskWriteBytes"], ["DiskReadOps", "DiskWriteOps"]]
+              %}
+              # ["CPUCreditUsage", "CPUCreditBalance"], ["CPUSurplusCreditBalance", "CPUSurplusCreditsCharged"] were deleted
```

Comment:
Let's remove them. This comment could be part of the commit message, describing that these metrics have not been included for this reason.

Reply:
Done, patch to merge with previous one about Refactoring Head Node metrics